src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractValidationServiceImpl.java

```diff
@@ -148,32 +174,44 @@ private Person findPerson(String employeeId) {
         return null;
     }
     
-    @Override
+    @Deprecated
     public boolean validateConcurStandardAccountingExtractDetailLine(ConcurStandardAccountingExtractDetailLine line) {
-        boolean valid = validateReportId(line.getReportId());
-        valid = validateAccountingLine(line) && valid;
-        valid = validateEmployeeId(line.getEmployeeId()) && valid;
+        ConcurStandardAccountingExtractBatchReportData reportData = new ConcurStandardAccountingExtractBatchReportData();
+        return validateConcurStandardAccountingExtractDetailLine(line, reportData);
+    }
+
+    @Override
+    public boolean validateConcurStandardAccountingExtractDetailLine(ConcurStandardAccountingExtractDetailLine line, ConcurStandardAccountingExtractBatchReportData reportData) {
+        boolean valid = validateReportId(line, reportData);
+        valid = validateAccountingLine(line, reportData) && valid;
+        valid = validateEmployeeId(line, reportData) && valid;
         return valid;
     }
     
-    private boolean validateReportId(String reportId) {
-        boolean valid = StringUtils.isNotBlank(reportId);
+    private boolean validateReportId(ConcurStandardAccountingExtractDetailLine line, ConcurStandardAccountingExtractBatchReportData reportData) {
+        boolean valid = StringUtils.isNotBlank(line.getReportId());
         if (valid) {
-            LOG.debug("validateReportId, found a valid report ID: " + reportId);
+            LOG.debug("validateReportId, found a valid report ID: " + line.getReportId());
         } else {
-            LOG.error("validateReportId, no valid report ID");
+            String validationError = "No valid report ID.";
+            reportData.addValidationErrorFileLine(new ConcurBatchReportLineValidationErrorItem(line.getReportId(), line.getEmployeeId(), line.getEmployeeLastName(), line.getEmployeeFirstName(), line.getEmployeeMiddleInitital(), validationError));
+            LOG.error("validateReportId, " + validationError);
         }
         return valid;
     }
 
-    private boolean validateAccountingLine(ConcurStandardAccountingExtractDetailLine line) {
+    private boolean validateAccountingLine(ConcurStandardAccountingExtractDetailLine line, ConcurStandardAccountingExtractBatchReportData reportData) {
         reportErrorsWithOriginalAccountingDetails(line);
```

Comment:
since there are reports and logs here, it would be clearer to rename this function to logErrorsWithOriginalAccountingDetails